File: wireframe/src/main/java/ru/nsu/e/shelbogashev/wireframe/Application.java
package ru.nsu.e.shelbogashev.wireframe;

import com.formdev.flatlaf.themes.FlatMacLightLaf;
import ru.nsu.e.shelbogashev.wireframe.bsplineeditor.BSplineEditor;

public class Application {
    public static void main(String[] args) {
        FlatMacLightLaf.setup();
        new BSplineEditor();
    }
}

File: wireframe/src/main/java/ru/nsu/e/shelbogashev/wireframe/Application.kt
package ru.nsu.e.shelbogashev.wireframe

import com.formdev.flatlaf.themes.FlatMacLightLaf
import ru.nsu.e.shelbogashev.wireframe.bsplineeditor.BSplineEditor

object Application {
    @JvmStatic
    fun main(args: Array<String>) {
        FlatMacLightLaf.setup()
        BSplineEditor()
    }
}
